src/Avalonia.Controls/TopLevel.cs

```diff
@@ -189,25 +188,27 @@ IStyleHost IStyleHost.StylingParent
         /// <inheritdoc/>
         protected virtual IRenderTarget CreateRenderTarget()
         {
+            if(PlatformImpl == null)
+                throw new InvalidOperationException("PlatformImpl isn't available");
```

Comment:
Could we change this to "Cannot create render target: PlatformImpl is null" or something? "isn't available" is a bit ambiguous.